src/styles/styles.js

```diff
@@ -742,7 +742,6 @@ const styles = {
     },
 
     sidebarAvatar: {
-        backgroundColor: themeColors.icon,
```

Comment:
Why was this change made?

Reply:
1) Desktop version doesn’t support require png/jpg images from assets, and we decided to use SVG Icon for default avatar.But icon is smaller then image and we put ‘100%’ value to Icon width and height to envelop parent View of Avatar and it’s look pretty good. React-Native engine calculate values of Avatar view and sent to Icon component numbers for width and height props.


2) This margin is placed for correct position of default avatar Icon in chats.


3) This style property gives gray color for background of Avatar View in some places like Header. If it’s transparent icon or avatar image, gray backround will be visible. So if we put default icon in offline mode, we don’t have cases when Avatar is not displayed, when there is empty space, so it seems that gray background will spoil our transparent default icon or some transprent avatars.

4) Regarding CLA, do we need to send addition comment for this? Because we typed the comment in the commit of PR.